LoopMusic/Source/ViewControllers/Settings/TrackSettingsViewController.swift

```diff
@@ -17,6 +20,29 @@ class TrackSettingsViewController: BaseSettingsSectionViewController {
         MusicPlayer.player.volumeMultiplier = Double(sender.value)
     }
     
+    /// Automatically normalizes the relative volume setting.
+    @IBAction func normalizeVolume() {
+        // Compute the track's intrinsic average volume.
+        var audioData = MusicPlayer.player.audioData
+        let framerateReductionLimit: Int = Int(round(MusicSettings.settings.frameRateReductionLimit))
+        let lengthLimit: Int = Int(MusicSettings.settings.trackLengthLimit)
+        let intrinsicVolume = Double(calcAvgVolumeFromBufferFormat(&audioData, framerateReductionLimit, lengthLimit))
+
+        if let text = volumeNormalizationLevelField.text, let normalizationLevel = Double(text) {
```

Comment:
This can use the value from `MusicSettings` instead of re-parsing the text field.

Reply:
Resolved in 23965ef